packages/core/src/browser/frontend-application.ts

```diff
@@ -157,17 +157,30 @@ export class FrontendApplication {
         return startupElements.length === 0 ? undefined : startupElements[0] as HTMLElement;
     }
 
+    protected inComposition = false;
```

Comment:
please add a comment to this PR that is a temporary fix and should be removed and retested with newer version of Monaco, that we don't forget

Reply:
I see. I added some comments for clarifying the code is a hotfix.